firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/LoginsHelper.js

```diff
@@ -578,6 +578,29 @@ window.__firefox__.includeOnce("LoginsHelper", function() {
     },
   }
 
+  // define the field types for focus events
+  const FocusFieldType = {
+    username: "username",
+    password: "password"
+  };
+
+  function onFocusIn(event) {
+    const form = event.target?.form;
+    if (!form) {
+      return;
+    }
+
+    const [username, password] = LoginManagerContent._getFormFields(form, false);
+    if (username && password) {
```

Comment:
Let's allow `password` only forms as well since `_getFormFields` supports it. 
```suggestion
    if (password) {
```